### ISSUE TITLE
Лишние типы

### ISSUE THREAD

**Issue by casa7x** [AnnLuschik/lesson-16-s#2]
https://github.com/AnnLuschik/lesson-16-s/blob/0a6d40f740e09dd1ba9f486af62daff2e3dc99e2/task4.js#L25

Здесь нету типов объекта и undefined, так что делать было не обязательно)

**Comment by AnnLuschik** [comment]
Да, но я добавила их, потому что в задании было сказано, что функции должны быть универсальными.
